docs/sources/configure/_index.md

```diff
@@ -1,8 +1,11 @@
 ---
-description: Describes parameters used to configure Grafana Loki.
-menuTitle: Configuration parameters
 title: Grafana Loki configuration parameters
-weight: 500
+menuTitle: Configuration parameters
+description: Configuration reference for the parameters used to configure Grafana Loki.
+aliases: 
+  - ../configuration
```

Comment:
This looks to be correctly referring to the previous location. To be certain, I would run `make docs` and navigate to the old page location and confirm that you get redirected to the new page.